setup.py

```diff
@@ -1,26 +1,31 @@
+import glob
```

Comment:
This file can be deleted entirely no?

Reply:
Seemingly. It took some adjusting but seems to be working now.